# HG changeset patch
# User Rik <rik@octave.org>
# Date 1550861634 28800
#      Fri Feb 22 10:53:54 2019 -0800
# Node ID 1218f6fa38569663c940f5eac8f387819ea3dd86
# Parent  8cad08868ffd907caf9fe309902515391f2fb221
ismethod.m: Add ';' to BIST test in cset 52f42a818717.

* ismethod.m: Add semicolon to assert statement within a %!test block.

diff --git a/scripts/miscellaneous/ismethod.m b/scripts/miscellaneous/ismethod.m
--- a/scripts/miscellaneous/ismethod.m
+++ b/scripts/miscellaneous/ismethod.m
@@ -40,12 +40,12 @@ function retval = ismethod (obj, method)
 
   method_list = methods (obj);
 
   retval = ismember (method, method_list);
 
 endfunction
 
 %!testif HAVE_JAVA; usejava ("jvm")
-%! assert (ismethod (javaObject ("java.lang.String", "Yo"), "hashCode"))
+%! assert (ismethod (javaObject ("java.lang.String", "Yo"), "hashCode"));
 
 %!assert (ismethod (ftp (), "ascii"))
 %!assert (ismethod (inputParser (), "addRequired"))
